lib/compat/wordpress-6.6/fonts/class-gutenberg-rest-font-faces-controller.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/**
+ * REST API: WP_REST_Font_Faces_Controller class
+ *
+ * @package WordPress
+ * @subpackage REST_API
+ * @since 6.5.0
```

Comment:
```suggestion
 * @since 6.6.0
```

I'm guessing this version number may need to be updated for 6.6?